Group entity - implementation with GroupRepository (JDP200101-13)

deseff opened the pull request.

deseff committed:
Group entity - implementation with GroupRepository (JDP200101-13)

Olafbb reviewed (commented):
Według mnie wszystko gra. Jedynie to kwestia adnotacji, czyli musimy się ugadać czy robimy je przy akcesorach czy przy polach 👍

Skorpien commented on src/main/java/com/kodilla/ecommercee/domain/Group.java:
```diff
@@ -0,0 +1,33 @@
+@Entity
+@Table(name = "products_groups")
+@Data
+@AllArgsConstructor
```
sorki, że ja się  tutaj wtrącę, ale wydaje mi się, że użycie @AllArgsConstructor może być tu trochę zgubne. Wydaje mi się, że wtedy do konstruktora wrzuca też private List<Product> products.
Tak więc masz albo bezparametrowy, albo 3 parametrowy (Long id, String name, List<Product> products), a tego trzeciego nie wiem czy jest sens w konstruktor wrzucać ;)

deseff replied:
Słuszna uwaga.

MARudnicki commented on build.gradle:
```diff
@@ -29,4 +29,6 @@ dependencies {
     runtimeOnly('com.h2database:h2')
     testImplementation('org.springframework.boot:spring-boot-starter-test')
+    compile('mysql:mysql-connector-java')
+    compileOnly('org.projectlombok:lombok')
```
implementation oraz compile znaczy to samo. Warto trzymac sie jednek skladni
https://stackoverflow.com/a/44493379

deseff replied:
Zmienione

deseff committed:
Group entity - implementation with GroupRepository (JDP200101-13)
- syntax unification in build.gradle
- class Group:
  - removing @AllArgsConstructor
  - adding constructor with id and name

deseff committed:
Group entity - implementation with GroupRepository (JDP200101-13)
- adjusting build.gradle to master to resolve conflicts
